apps/server/src/v1/monitor.ts

```diff
@@ -31,6 +31,14 @@ const ParamsSchema = z.object({
     }),
 });
 
+const isoDate = z
+  .preprocess((val) => {
```

Comment:
this should be exported in a `utils.ts` and import it where ever we need it

Reply:
sure